README_TTN.md

```diff
@@ -103,6 +107,134 @@ Details are available [here](https://blog.squix.org/2017/07/seeeduino-lora-gps-g
 Compiling the OTAA sketch should make the Seeeduino connect to the Gateway and
 send messages to the TTN.
 
+## Receive data from TTN into IoT-LoRaWan Agent
+
+In order to receive data from TTN the IoT-Lorawan Agent should
+subscribe to the applications on the TTN platform.
+
+The code below is the comand, already configured, that we use in order to connect.
+
+Two keys, one for each application are generated in order to
+connect and used as password.
+Remember to remove the security code.
+
+```bash
+curl --location --request POST 'localhost:4042/iot/services' \
+--header 'fiware-service: openiot' \
+--header 'fiware-servicePath: /' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+  "services": [
+    {
+      "entity_type": "waste",
+      "apikey": "",
+      "resource": "8CF957200005727C",
+      "attributes": [
+        {
+          "object_id": "temperature_1",
+          "name": "temperature",
+          "type": "Number"
+        },
+        {
+          "object_id": "relative_humidity_1",
+          "name": "relative_humidity",
+          "type": "Number"
+          },
+          {
+            "object_id": "digital_in_1",
+            "name": "flame_detected",
```

Comment:
the thing is a bit more complex than this, you need to compute an attribute `status` based on the different sensor values.

ok. Container is where it must be and stands properly.
lidOpen. Container's lid has been opened and not closed after a certain amount of time.
dropped. Container has been dropped for some reason.
moved. Container has been moved from its regular position and has not come back.
vandalized. Container has been damaged or destroyed due to vandalism.
burning. Container is burning and an immediate action has to be taken.
unknown. The status of the container is not known to the system.

Reply:
so you need an named "status" an use expressions to compute the value. anyhow, check the old code:
https://gitlab.com/orchestracities/iot/-/blob/master/diorama/src/waste/create_device.sh

Reply:
https://gitlab.com/orchestracities/iot/-/blob/master/diorama/src/waste/waste.ino#L143

if you want to compute the status in arduino, if not you can use a jexl expression here (may be more complex)